bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,69 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+import yaml
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<filename>\S+):(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL})
+class HamlLintBear:
+    """
+    Checks code using ``haml-lint`` for
+    HAML-specific style and lint checks
+    to ensure clean and readable HAML code.
+    """
+    LANGUAGES = {'Haml'}
+    REQUIREMENTS = {GemRequirement('haml_lint', '0.26.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    SEE_MORE = 'https://github.com/brigade/haml-lint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        line_length_enable: bool=True,
+                        line_length_max: int=80,
+                        trailing_whitespace_enabled: bool=True):
+        """
+        :param line_length_enable:
+            Use this rule to check whether maximum line length linter is
+            enabled or not.
+        :param line_length_max:
+            Maximum number of characters for a line, the newline character
+            being excluded.
+        :param trailing_whitespace_enabled:
+            Use this rule to check whether trailing whitespace linter is
+            enabled or not.
+        """
+        hamllint_configs = {
+            'linters': {
+                'LineLength': {
+                    'enabled': line_length_enable,
+                    'max': line_length_max,
+                },
+                'TrailingWhitespace': {
+                    'enabled': trailing_whitespace_enabled,
+                },
+            },
+        }
+
```

Comment:
I can see many linters [here](https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md) out of which very few are included. It would be nice if we could make the bear super configurable :+1: